src/dotnet/ToolPackage/IToolPackageRepository.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.EnvironmentAbstractions;
+
+namespace Microsoft.DotNet.ToolPackage
+{
+    internal interface IToolPackageRepository
```

Comment:
Maybe store? Repository should have install on it

Reply:
Agreed.  I'll rename.

Reply:
Renamed in upcoming push.